Error from Jenkins indicates that while this is the correct change to update the go-mod-core-contracts reference, the code was never built locally. The import path for the logging and related references in the code needs to be fixed.
```
11:02:54 go: finding github.com/edgexfoundry/go-mod-core-contracts/clients/logging latest
11:02:54 go: finding github.com/edgexfoundry/go-mod-core-contracts/clients latest
11:02:54 main.go:32:2: unknown import path "github.com/edgexfoundry/go-mod-core-contracts/clients/logging": cannot find module providing package github.com/edgexfoundry/go-mod-core-contracts/clients/logging
11:02:54 make: *** [build_worker] Error 1
11:02:54 Build step 'Execute shell' marked build as failure
```

https://github.com/edgexfoundry/go-mod-core-contracts/blob/c482d7c75c33440653bf0ca5836cf39eac998138/clients/logger/logger.go#L19

--- a/core/main.go
+++ b/core/main.go
@@ -29,16 +29,11 @@ import (
 	"os"
 	"time"
 
-	logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
+	logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
 )
 
 var debug = false
-var lc = CreateLogging()
-
-// CreateLogging Logger functionality
-func CreateLogging() logger.LoggingClient {
-	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")), logger.InfoLog)
-}
+var lc logger.LoggingClient
 
 func main() {
 

--- a/core/main.go
+++ b/core/main.go
@@ -30,10 +30,17 @@ import (
 	"time"
 
 	logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	model "github.com/edgexfoundry/go-mod-core-contracts/models"
 )
 
 var debug = false
-var lc logger.LoggingClient
+var lc = CreateLogging()
+
+// CreateLogging Logger functionality
+func CreateLogging() logger.LoggingClient {
+	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")), model.InfoLog)
+}
+
 
 func main() {
 

